spec/forms/guide_form_spec.rb

```diff
@@ -404,3 +418,11 @@
     expect(guide_form.to_param).to eq("5")
   end
 end
+
+RSpec.describe GuideForm, "#slug_prefix" do
+  it "is /service-manual" do
```

Comment:
Sorry, probably should've pointed this one out as well (on the returns / is a front)